romanNumerals.js

```diff
@@ -0,0 +1,89 @@
+module.exports = {convertToOldRoman, convertToNewRoman}
+
+function convertToNewRoman(num){
+  var roman = ""
+  while(num >= 900){
+    if(num < 1000){
```

Comment:
You could have used the same pattern you used for `convertToOldRoman` if you treated `CM` as 900. That being said can you thing of a more reusable way to do this? Maybe an object that maps the roman number to the number i.e `{ 'M' : 1000, 'CM': 900, ...}` and looping through that